docs/migration_guides/opflow_migration.rst

```diff
@@ -0,0 +1,1677 @@
+#######################
+Opflow Migration Guide
+#######################
+
+TL;DR
+=====
+The new :mod:`~qiskit.primitives`, in combination with the :mod:`~qiskit.quantum_info` module, have superseded
+functionality of :mod:`~qiskit.opflow`. Thus, the latter is being deprecated.
+
+In this migration guide, you will find instructions and code examples for how to migrate your code based on
+the :mod:`~qiskit.opflow` module to the :mod:`~qiskit.primitives` and :mod:`~qiskit.quantum_info` modules.
+
+.. note::
+
+    The use of :mod:`~qiskit.opflow` was tightly coupled to the :class:`~qiskit.utils.QuantumInstance` class, which
+    is also being deprecated. For more information on migrating the :class:`~qiskit.utils.QuantumInstance`, please
+    read the `quantum instance migration guide <http://qisk.it/qi_migration>`_.
+
+.. _attention_primitives:
+
+..  attention::
+
+    Most references to the :class:`qiskit.primitives.Sampler` or :class:`qiskit.primitives.Estimator` in this guide
+    can be replaced with instances of the:
+
+    - Aer primitives (:class:`qiskit_aer.primitives.Sampler`, :class:`qiskit_aer.primitives.Estimator`)
```

Comment:
As mentioned in the comment in `conf.py`, you will need to add `qiskit_aer` to intersphinx mapping for these links to be generated.

Reply:
Counting on https://github.com/Qiskit/qiskit-terra/pull/9716 to do that